python_ta/typecheck/base.py

```diff
@@ -127,6 +175,14 @@ def op_to_dunder_unary(op):
         return op
 
 
+def _correct_article(noun : str):
```

Comment:
If you're going to put in type annotations (good!) then put in a return type as well.